sources/settings-migrations/v1.22.0/bootstrap-commands-services-cfg-v0-1-0/src/main.rs

```diff
@@ -0,0 +1,18 @@
+use migration_helpers::common_migrations::{ListReplacement, ReplaceListsMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+fn run() -> Result<()> {
+    migrate(ReplaceListsMigration(vec![ListReplacement {
+        setting: "services.bootstrap-commands.configuration-files",
```

Comment:
This content will be removed on downgrade by the `AddPrefixesMigration` that covers `services.bootstrap-commands`, so you don't need this migration.